models/commentsModels.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection");
+
+exports.fetchCommentsByArticleIdFromDb = (id) => {
+  const queryStr = `
+        SELECT 
+        comment_id,
+        votes,
+        created_at,
+        author,
+        body
+        FROM comments
+        WHERE article_id = $1
+    `;
+
+  return db.query(queryStr, [id]).then((data) => {
+    const comments = data.rows;
+    if (comments.length === 0) {

```

Comment:
What if the article exists but there are no comments? Is this still an error?

Reply:
sorted :)